README.md

```diff
@@ -34,3 +84,21 @@ Please read the [contribution guidelines](https://github.com/opencv/opencv/wiki/
 * [Follow OpenCV on Mastodon](http://mastodon.social/@opencv) in the Fediverse
 * [Follow OpenCV on Twitter](https://twitter.com/opencvlive)
 * [OpenCV.ai](https://opencv.ai): Computer Vision and AI development services from the OpenCV team.
+
+---
+
+## Support OpenCV  
+
+Your contributions help us maintain and improve OpenCV. Consider supporting us via [donations](https://opencv.org/support/).  
+
+---
+
+### License  
+
+OpenCV is released under the **3-clause BSD license**. See the [LICENSE](https://github.com/opencv/opencv/blob/master/LICENSE) file for details.  
```

Comment:
I think it is better to reference https://opencv.org/license/ and update. This is not correct strictly. (Line 4 is same as here.)

Reply:
This is only my opinion. But currently OpenCV library is released under Apache 2 License.

[1] Quote from https://opencv.org/license

```
> The OpenCV library is licensed under two different licenses depending on the version of the library.
> OpenCV 4.5.0 and higher versions are licensed under the [Apache 2 License](https://github.com/opencv/opencv/blob/master/LICENSE). 
> See the [LICENSE](https://opencv.org/license/) for details.  
```

[2] Or simply link only.

```
> See the [LICENSE](https://opencv.org/license/).  
```

Reply:
@Kumataro I've addressed the feedback. Please review the changes.